content/doc/tutorials/using-jenkinsfile-runner-github-action-to-build-jenkins-pipeline.adoc

```diff
@@ -0,0 +1,308 @@
+The following is an example Jenkinsfile, where the name of this Jenkinsfile is “Jenkinsfile”:
+[source,groovy]
+----
+pipeline {
+    agent any
+    stages {
+        stage('hello') {
+            steps {
+                sh 'echo Hello Jenkins!'
+            }
+        }
+    }
+}
+----
+
+=== Create a plugin list file
+
+In this section, you will need to create a plugin list file which specifies which plugins you will need to install in the ephemeral Jenkins instance. 
+You can check link:https://github.com/jenkinsci/plugin-installation-manager-tool#plugin-input-format[the valid plugin input format]. 
+As we don’t have to install any extra plugins, you can create an empty plugin list file called “plugins.txt”.
+
+=== Create a workflow definition in the container
+
+In this section, you’ll learn how to create a workflow definition so you can run Jenkins pipeline in the GitHub Actions.
+
+* In your local GitHub repository, prepare a workflow definition file ci.yml (the file name doesn’t matter but it should be a YAML file) under the “.github/workflows” directory.
+* Create the name of your workflow and the event that triggers your workflow. 
+You can check link:https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#available-events[the available events] that trigger workflows. 
+The “push” event will trigger the workflow when you push a commit or tag.
+[source,yaml]
+----
+name: Java CI
+on: [push]
+----
+* Use a Ubuntu runner for the job.
+[source,yaml]
+----
+jobs:
+  job-name:
+    runs-on: ubuntu-latest
+----
+* (Optional) If you want to use `jfr-container-action` later, you need to declare using the `ghcr.io/jenkinsci/jenkinsfile-runner:master` or any image extending it. 
+If you choose to use `jfr-static-image-action`, you can skip this step.
+[source,yaml]
+----
+jobs:
+  job-name:
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/jenkinsci/jenkinsfile-runner:master
+----
+* Call the `actions/checkout@v2` to pull your codes into the runner. 
+“Call” means “uses” in the workflow definition specifically. 
+You can check the details about the “uses” keyword.
+[source,yaml]
+----
+- uses: actions/checkout@v2
+----
+* Call the Jenkinsfile-runner actions.
+** If you use `jfr-container-action`, you need to call `jenkinsci/jfr-container-action@master` and give necessary inputs.
+[source,yaml]
+----
+uses:
+  jenkinsci/jfr-container-action@master
+with:
+  command: run
+  jenkinsfile: Jenkinsfile
+  pluginstxt: plugins.txt
+----
+** If you use `jfr-static-image-action`, you need to call `jenkinsci/jfr-static-image-action@master` and give necessary inputs.
+[source,yaml]
+----
+uses:
+  jenkinsci/jfr-static-image-action@master
+with:
+  command: run
+  jenkinsfile: Jenkinsfile
+  pluginstxt: plugins.txt
+----
+
+You can check the full example workflow definition for link:http://jfr-static-image-action[jfr-container-action] and link:http://jfr-static-image-action[jfr-static-image-action]. 
```

Comment:
```suggestion
Refer to the full example workflow definition for link:http://jfr-static-image-action[jfr-container-action] or link:http://jfr-static-image-action[jfr-static-image-action] to verify your work. 
```
These links do not seem to lead anywhere, so they should either be removed as links, or the proper links need to be provided here. Additionally, there would need to be extra wording here to make the sentence readable.

Reply:
Sorry, It seems that I didn't copy the url correctly. Let me update it later.

Reply:
I just updated the URLs in the latest commit.